main/src/main/java/org/mobilitydata/gtfsvalidator/table/GtfsNetworkSchema.java

```diff
@@ -0,0 +1,15 @@
+package org.mobilitydata.gtfsvalidator.table;
+
+import org.mobilitydata.gtfsvalidator.annotation.*;
+
+@GtfsTable("agency.txt")
```

Comment:
Should be `network.txt`, no?

Reply:
The correct filename is `networks.txt`, `https://gtfs.org/schedule/reference/#networkstxt`

Reply:
Good catch!